### PR TITLE
make answer/submission distinction clear in variable names

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: core/services/eth/contracts/FluxAggregator.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func (fa *fluxAggregator) SubscribeToLogs(listener ethsvc.LogListener) (connecte
 type FluxAggregatorRoundState struct {
 	ReportableRoundID uint32   `abi:"_roundId"`
 	EligibleToSubmit  bool     `abi:"_eligibleToSubmit"`
-	LatestAnswer      *big.Int `abi:"_latestRoundAnswer"`
+	LatestAnswer      *big.Int `abi:"_latestResponse"`
```
Comment (j16r) [inline core/services/eth/contracts/FluxAggregator.go:78]: Isn't `latestRoundAnswer` more consistent with the other docs etc? Makes much more sense to me.

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol [hunk 2]
```diff
@@ -113,7 +113,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
     address newAdmin
   );
   event SubmissionReceived(
-    int256 indexed answer,
+    int256 indexed response,
```
Comment (j16r) [inline evm-contracts/src/v0.6/dev/FluxAggregator.sol:116]: Should this be something like "value" ? I'm confused by response in this context, where is the request?

File: core/services/eth/contracts/FluxAggregator.go (hunk 1 above)
Comment (se3000) [reply core/services/eth/contracts/FluxAggregator.go:78]: In docs so far we've tried to make a distinction. Response is a single oracle's current view. Submission is the tuple of Round ID and Response. Answer is the aggregated value of many oracles.

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol (hunk 2 above)
Comment (se3000) [reply evm-contracts/src/v0.6/dev/FluxAggregator.sol:116]: Just trying to make it consistent.

Comment (se3000) [reply evm-contracts/src/v0.6/dev/FluxAggregator.sol]: "Value" is a little overloaded and vague. Went with "submission," the whole tuple distinction felt a bit like hair splitting; it's a submission for a round.

**Pull request merged**